docs/release_notes/release_notes.md

```diff
@@ -8,42 +8,53 @@ has_children: true
 
 # Release Notes
 
-## Version 2.1.0 (May 2022)
+## Version 2.2.0 (Jun 2022)
 
 
+* ### New dashboard `Windows Patch` for the windows update events.
+
+* ### VPN dashboard improvement
+    * Added `Successful Session` panel.
+    * Added drilldown on `Connected Workforce by Location` map
+
 * ### Ransomware alert improvement
-    * Added filters for paths to reduce false positives for both report `Ransomware - Calculate UpperBound for Spike in File Writes` and alert `Ransomware - Spike in File Writes`.
+    * Enhanced filters for paths to reduce false positives for both report `Ransomware - Calculate UpperBound for Spike in File Writes` and alert `Ransomware - Spike in File Writes`.
 
-    * Increased minimun file write limit from 20 to 3000 to reduce the false positive.
+* ### Network Reports dashboard improvement 
+    * Added drilldown for `Port Scanning Attempts` map in `Network Reports` dashboard
 
-    * Added `top5_file_extension`, `avg`, and `stdev` field in the `Ransomware - Spike in File Writes` alert.
+* ### Linux/Unix dashboard improvement
+    * Added time field in the `Success Login by Host, Users` and `Failed Login by Host, Users` panels in the `Linux/Unix` dashboard
+    * Removed the `Open Ports` panel as `Listening Ports on Host` panel provides same details with more fields.
 
-    * Added `parent_process_path` field in the `Ransomware - Endpoint Compromise - Fake Windows Processes` alert.
+* ### Windows Reports dashboard improvement
+    * Added `Listening Ports on Host` panel in the `Windows Reports` dashboard. For the data collection, User need to enable the `win_listening_ports` scripted input.
 
-* ### Palo Alto Firewall System alert and dashboard improvement
-    * Excluded the License related events from the `Palo Alto High System Alert` alert and `System Events` panel in the Palo Alto dashboard.
-    * Added a new `License Events` panel to show all Palo Alto license-related events.
+* ### Office 365 alert and dashboard improvement
+    * Updated the alerts and dashboard to use the new o365:service:healthIssue sourcetype (as the o365:service:status sourcetype is retired)
```

Comment:
Updated the alerts and dashboard to use the new `o365:service:healthIssue` sourcetype (as the` o365:service:status sourcetype` is retired by the Add-on)